rayon-futures/src/lib.rs

```diff
@@ -2,6 +2,8 @@
 //!
 //! See `README.md` for details.
 
+#![doc(html_root_url = "https://docs.rs/rayon-futures/0")]
```

Comment:
Let's make this `0.1` so it's in line with what's in the `Cargo.toml`, even if that hasn't been released yet.